circle.yml

```diff
@@ -17,8 +17,8 @@ notify:
 machine:
   # Overwrite these ENV variables in parametrized (manual/API) builds
   environment:
-    DISTROS: "wheezy jessie trusty el6 el7"
-    NOTESTS: "el7"
+    DISTROS: "wheezy jessie trusty xenial el6 el7"
+    NOTESTS: "xenial el7"
```

Comment:
Did you figure out why el7 tests were not run? Were they breaking and we just YOLOed?